scruml/__init__.py

```diff
@@ -1,12 +1,26 @@
 # ScrUML
 # __init__.py
 # Team JJARS
+import sys
+from typing import List
+
 from scruml import uml_context_cli
+from scruml import uml_context_gui
 
 
-def main() -> None:
+def main(argv: List[str] = sys.argv) -> None:
     """Act as primary entry point for the program."""
-    uml_context_cli.activate()
+    if len(argv) == 1 or (len(argv) == 2 and argv[1] == "--gui"):
+        # If no arguments were passed or "--gui" was specified, run in GUI mode
+        uml_context_gui.activate()
```

Comment:
We might want launching the GUI to be the default behavior. A reasonable expectation by the user might be that typing in "scruml" would open the program instead of printing a usage message.

Reply:
Additionally, we could have launching the CLI be the default behavior until the experimental GUI is finished, and swap behaviors near the end of this sprint